### DISCUSSION TITLE
Shoonya v2.0

### DISCUSSION THREAD

**Discussion by aravinth** [AI4Bharat/Shoonya#17] [Announcements]
Any discussion related to the v2.0 features

**Comment by venkatsriraman** [comment]
Has this impacted - [demo.shoonya.ai4bharat.org](https://demo.shoonya.ai4bharat.org/) not able to access the same.

**Reply by aravinth** [reply]
Yes @venkatsriraman - demo instance was not being actively used. So we brought it down due to cost involved.
